src/main/java/io/github/mivek/parser/AbstractParser.java

```diff
@@ -158,7 +146,7 @@ protected boolean generalParse(final AbstractWeatherContainer pContainer, final
      * @param pParts the tokens of the event
      * @param index the RMK index in the event.
      */
-    protected void parseRMK(final AbstractWeatherContainer pContainer, final String[] pParts, final int index) {
+    void parseRMK(final AbstractWeatherContainer pContainer, final String[] pParts, final int index) {

```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Use explicit scoping instead of the default package private level](https://app.codacy.com/app/mivek/MetarParser/pullRequest?prid=4593741)